app/scripts/modules/core/presentation/navigation/pageNavigator.component.spec.ts

```diff
@@ -34,7 +34,7 @@ describe('Component: Page Navigator', () => {
     let dom = `
       <div class="container" style="height: 60px; overflow-y: scroll">
         <page-navigator scrollable-container=".container">`;
-    pages.forEach((p, index) => {
+    pages.forEach((_p, index) => {
       dom += `
           <page-section key="{{pages[${index}].key}}" label="{{pages[${index}].label}}" visible="pages[${index}].visible">
```

Comment:
Ahhhh.  No.  These are bound models within angular - you don't want to dereference them in the actual template string, but rather have angular dereferencing them.  Have to leave this as is.